### PR TITLE
fix:添加style变量赋值

### PR DESCRIPTION
**Pull request by Xppp0217**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

### What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/settings/styles/src/Main.vue [hunk 1]
```diff
@@ -146,29 +166,87 @@ export default {
...
+        border-radius: 8px;
+        width: 216px;
+        text-align: left;
+      }
+    }
+    .inline-bind-style {
+      :deep(.tiny-input__inner) {
+        width: 216px;
+        pointer-events: none;
+        background: #1476ff;
```
Comment (chilingling) [inline packages/settings/styles/src/Main.vue]: 需要兼容暗色模式

File: packages/settings/styles/src/Main.vue [hunk 2]
```diff
@@ -8,14 +8,32 @@
...
-    <meta-code-editor
-      :modelValue="state.styleContent"
-      title="Style 编辑"
-      button-text="编辑行内样式"
-      language="css"
-      single
-      @save="save(CSS_TYPE.Style, $event)"
-    />
+    <div class="line-style">
+      <span style="display: block; margin-top: 16px; margin-bottom: 8px; color: #595959; font-size: 12px">
```
Comment (chilingling) [inline packages/settings/styles/src/Main.vue]: 为什么要用行内样式？

File: packages/common/component/MetaBindVariable.vue [hunk 3]
```diff
@@ -386,7 +386,7 @@ export default {
     }
 
     const remove = () => {
-      emit('update:modelValue', '')
+      emit('update:modelValue', 'remove')
```
Comment (chilingling) [inline packages/common/component/MetaBindVariable.vue]: 这个改动对其他依赖该组件的功能会有影响吗？

**Review by chilingling — CHANGES_REQUESTED** [review]

File: packages/settings/styles/src/Main.vue [hunk 4]
```diff
@@ -146,29 +166,95 @@ export default {
...
+      background: var(--ti-lowcode-setting-style-input-bg);
+      color: var(--ti-lowcode-setting-style-input-font-color);
+      border-color: var(--ti-lowcode-setting-style-input-bg);
     }
   }
 }
+:deep(.svg-icon) {
+  margin-top: 5px;
+}
+}
```
Comment (chilingling) [inline packages/settings/styles/src/Main.vue:263]: 这里是多了一个分号还是格式化不对？

File: packages/common/component/MetaBindVariable.vue (hunk 3 above)
Comment (Xppp0217) [reply packages/common/component/MetaBindVariable.vue]: 目前看并无影响

File: packages/settings/styles/src/Main.vue (hunk 4 above)
Comment (Xppp0217) [reply packages/settings/styles/src/Main.vue:263]: 格式化问题

File: packages/common/component/MetaBindVariable.vue (hunk 3 above)
Comment (chilingling) [reply packages/common/component/MetaBindVariable.vue]: image

这个会影响组件绑定，复现逻辑：
1. 属性面板点击绑定属性，随便绑定一个变量
image

2.  绑定后再次点击绑定变量弹窗，点击移除绑定
image

移除绑定后会导致并没有清除绑定，而是出来 remove 值

File: packages/settings/styles/src/Main.vue [hunk 5]
```diff
@@ -146,29 +164,95 @@ export default {
...
+        schema.props.style = ''
+        currentSchema.props.style = ''
+        state.propertiesList = '编辑行内样式'
+        state.lineStyleDisable = true
+        addHistory()
+      }
+    }
+
+    watch(
+      () => {
```
Comment (chilingling) [inline packages/settings/styles/src/Main.vue]: 可以简化成 `() => getCurrentSchema()`

File: packages/settings/styles/src/Main.vue [hunk 6]
```diff
@@ -146,29 +164,101 @@ export default {
...
+        state.propertiesList = '编辑行内样式'
+        state.lineStyleDisable = true
+        addHistory()
+      }
+    }
+
+    watch(
+      () => getCurrentSchema(),
+      (val) => {
+        if (val?.props && val.props?.style?.value) {
```
Comment (chilingling) [inline packages/settings/styles/src/Main.vue]: `if (val?.props?.style?.value) {` 就可以了

**Review by chilingling — APPROVED** [review]

**Pull request merged**